lib/iris/tests/unit/experimental/ugrid/cf/test_CFUGridAuxiliaryCoordinateVariable.py

```diff
@@ -16,16 +16,18 @@
 
 import numpy as np
 
-from iris.experimental.ugrid import logger
-from iris.experimental.ugrid.cf import CFUGridAuxiliaryCoordinateVariable
+from iris.experimental.ugrid.cf import (
+    CFUGridAuxiliaryCoordinateVariable,
+    logger,
+)
 from iris.tests.unit.experimental.ugrid.cf.test_CFUGridReader import (
     netcdf_ugrid_variable,
 )
 
 
 def named_variable(name):
     # Don't need to worry about dimensions or dtype for these tests.
-    return netcdf_ugrid_variable(name, "", np.int)
+    return netcdf_ugrid_variable(name, "", int)
```

Comment:
The use of `np.int` as an alias for the builtin `int` was [deprecated](https://numpy.org/doc/stable/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated) in `numpy` v1.20.0